src/classes/PS_IntegrationServiceConfig.cls

```diff
@@ -51,6 +51,8 @@ public with sharing class PS_IntegrationServiceConfig {
     public static final String SFDO_USERNAME = 'sfdo.username';
     public static final String ELEVATE_SDK = 'elevateSDK';
     public static final String PRODUCTID = 'productId';
+    public static final String PURCHASE_ROLE = 'Payments.Purchase';
```

Comment:
These constants are not required after the latest changes.

Reply:
@RadSFDeveloper updated!